boring/src/ssl/mod.rs

```diff
@@ -2217,7 +2217,7 @@ impl SslContextRef {
     /// threads.
     #[cfg(not(feature = "fips"))]
     #[corresponds(SSL_CTX_set1_ech_keys)]
-    pub fn set_ech_keys(&self, keys: SslEchKeys) -> Result<(), ErrorStack> {
+    pub fn set_ech_keys(&self, keys: &SslEchKeys) -> Result<(), ErrorStack> {
```

Comment:
`SSL_CTX_set1_ech_keys` doesn't [document](https://commondatastorage.googleapis.com/chromium-boringssl-docs/ssl.h.html#SSL_CTX_set1_ech_keys) if it takes ownership or not. I had to dig the source to see it bumps refcount.